recipe/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "kaldi-native-fbank" %}
+{% set version = "1.17" %}
+
+# version is in format 1.X_Y_ZZZZZZ because the release is based on 1.17 from the main developer
+# but a commit was added from a fork from developer cbalioglu
+package:
+  name: {{ name|lower }}
+  version: {{ version }}+1_740f131
```

Comment:
It's not as usual to see this versioning based on a fork commit. It can confuse users and our future selves.
As we only apply a [patch](https://github.com/cbalioglu/kaldi-native-fbank/commit/740f1313a71b618418154a143506eb501a4dfd12) why do we change the version? We download the package from PYPi not from a forked repo.

Reply:
Because version **1.17** (without the patch) would not be compatible with `fairseq2`. This is a package for the snowflake channel, so **default** users would not use this package. For snowflake users this package would also have limited visibility because it would not be present on the official SF platform. I tried to document this choice as much as possible to not confuse ourselves in the future (assuming we will ever touch this).

Alternatively, if you don't like this scheme (which, admittedly, is usually used only when you're building a version between two tags), I could use another name scheme where I specify the name of the fork.
e.g.
```
name: {{ name|lower }}-cbalioglu
version: {{ version }}-1_740f131
```